assembl/scripts/lambda_create_db_aws_user.py

```diff
@@ -0,0 +1,38 @@
+import pg8000
+import boto3
+import os
+import base64
+import json
+
+
+def lambda_handler(input, ctx):
+    sm = boto3.client('secretsmanager')
+    response = sm.get_secret_value(SecretId='db_password_assembl')
+    if 'SecretString' in response:
+        info = response['SecretString']
+    else:
+        info = base64.b64decode(response['SecretBinary'])
+    info = json.loads(info)
+    database = input.get('database', os.environ.get('database', 'assembl'))
+    cnx = pg8000.core.Connection(
```

Comment:
This script is not loaded in `assembl`. The script is in fact managed by AWS Lambda. pg8000 is used because it's a dependency-less DBAPI implementation in python, which can be used easily in Lambda.